portfolio/src/main/webapp/script2.js

```diff
@@ -0,0 +1,7 @@
+async function getMessage() {
```

Comment:
Could you add a comment here explaining what the getMessage method does.

Eg: Fetches response from /hello and replaces text in the "greeting-container"